src/modules/transformers.ts

```diff
@@ -60,10 +60,12 @@ const getCombinedRolesForDomains = (
 };
 
 export const getAllUserRolesForDomain = (
-  { domains, roles }: Colony,
+  colony: Colony | undefined,
```

Comment:
Why is this or `undefined`?

Reply:
I added that due to these lines in the PermissionManagementDialog:

```
  const domainRoles = useTransformer(getAllUserRolesForDomain, [
    colonyData?.colony,
    selectedDomainId,
  ]);

  const directDomainRoles = useTransformer(getAllUserRolesForDomain, [
    colonyData?.colony,
    selectedDomainId,
    true,
  ]);
```

colonyData might be `undefined`. I saw that in `getUserRolesForDomain` the colony param can be undefined and in that case it returns an empty array, so I figured we could apply that same thing to `getAllUserRolesForDomain`